src/Symfony/Component/DependencyInjection/LazyProxy/PhpDumper/DumperInterface.php

```diff
@@ -38,7 +38,7 @@ public function isProxyCandidate(Definition $definition);
      *
      * @return string
      */
-    public function getProxyFactoryCode(Definition $definition, $id, $methodName);
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null);
```

Comment:
are you sure about that ? Implementors are allowed to add `= null` for their argument even if the interface does not have it (the opposite is false).